evm/contracts/Aggregator.sol

```diff
@@ -219,13 +221,61 @@ contract Aggregator is ChainlinkClient, Ownable {
     if (responseLength % 2 == 0) {
       int256 median1 = quickselect(answers[_answerId].responses, middleIndex);
       int256 median2 = quickselect(answers[_answerId].responses, middleIndex.add(1)); // quickselect is 1 indexed
-      currentAnswer = median1.add(median2) / 2; // signed integers are not supported by SafeMath
+      currentAnswerValue = median1.add(median2) / 2; // signed integers are not supported by SafeMath
     } else {
-      currentAnswer = quickselect(answers[_answerId].responses, middleIndex.add(1)); // quickselect is 1 indexed
+      currentAnswerValue = quickselect(answers[_answerId].responses, middleIndex.add(1)); // quickselect is 1 indexed
```

Comment:
If we store currentAnswerValue in another in-memory variable for the above two instances and set it below with the other storage variables, we can save on the gas cost of reading it from storage which goes up 4x to 800 soon.